Argo/Globals/maybe.swift

```diff
@@ -0,0 +1,6 @@
+public func maybe<A, B>(b: B, f: A -> B, a: A?) -> B {
```

Comment:
I really (really) dislike the introduction of this function, so if we can get rid of it, that would be great.